robottelo.properties.sample

```diff
@@ -70,7 +70,7 @@ ssh_key=
 # [ldap]
 # hostname=
 # username=
-# passwd=
+# password=
```

Comment:
Will handle it on next rebase. Let's introduce this change here since the new search will be on hold for a while.